website/frontend/static/css/styles.css

```diff
@@ -5520,3 +5520,11 @@ pre {
   white-space: nowrap;
   border-radius: 4px;
 }
+
+.ddm{
+  display: none;  
+}
+
+.download-btn:hover .ddm{
+  display: block;
```

Comment:
Why the extra `ddm` class (stands for dropdown menu?), `.download-btn:hover .dropdown-menu` would be just fine and nicer to read. It also looks weird in the HTML (this element is a dropdown-menu and also a ddm!?)